docs/patterns/navigation.md

```diff
@@ -14,12 +14,16 @@ displayed horizontally on larger screens.
 
 <div class="p-notification--information">
   <p class="p-notification__response">
-    <span class="p-notification__status">Note:</span>By default, the width of the navigation is constrained to <code>$grid-max-width</code>. To make the nav full width, replace <code>.p-navigation__row</code> with <code>.p-navigation__row--full-width</code>.
+    <span class="p-notification__status">Note:</span>By default, the width of the navigation is constrained to <code>$grid-max-width</code>. To make the navigation full width, replace <code>.p-navigation__row</code> with <code>.p-navigation__row--full-width</code>.
   </p>
 </div>
 
-The background color of a navigation pattern can be set via the
-`$color-navigation-background` variable.
+The navigation pattern is one of the first patterns to implement the new theming architecture in Vanilla. The default is light. But, to switch to a dark navigation, you can either:
+
+- Change the value of the `nav` key in the `$theme-default--dark` map (located in `_settings_themes.scss`) to `true`
```

Comment:
This is how to change it inside Vanilla but not how you should override it in a site.